aquacomputer_d5next.c

```diff
@@ -842,6 +916,11 @@ static umode_t aqc_is_visible(const void *data, enum hwmon_sensor_types type, u3
 	case hwmon_curr:
 		switch (priv->kind) {
 		case aquastreamxt:
+			/* Special case to support pump and fan current */
+			if (channel < 2)
+				return 0444;
+			break;
+		case aquastreamult:
```

Comment:
You might have mixed up Aquastream XT and Ultimate here.

Reply:
Embarrassing mistakes, thanks for noticing!